schedule.yml

```diff
@@ -31,6 +31,38 @@ Thursday 15th:
     '15:00': talks/is-my-robot-butler-going-to-steal-my-job
     '15:30': talks/where-am-i-what-am-i-doing-here-motion-detection-and-sensor-fusion-on-wearables-and-microbit-with-python
     '16:00': Close
+  CU Room A:
+    '08:00': Registration
+    '09:15': ''
+    '09:30': workshops/django-vs-flask-vs-pyramid-which-to-chose
+    '10:00': workshops/django-vs-flask-vs-pyramid-which-to-chose
+    '10:30': workshops/django-vs-flask-vs-pyramid-which-to-chose
+    '11:00': Break
+    '11:30': workshops/natural-language-processing-in-10-lines-of-code
+    '12:00': workshops/natural-language-processing-in-10-lines-of-code
+    '12:30': workshops/natural-language-processing-in-10-lines-of-code
+    '13:00': Lunch
+    '14:00': Building a Raspberry Pi supercomputer in a suitcase
```

Comment:
Assume this is text rather than a link to a file because we don’t have a description for this yet? (I can’t see one in this pull request, anyway.)